README.md

```diff
@@ -1,28 +1,28 @@
 # Sensu Docs Site
 
-1. [Project Setup](#project-setup)
+Welcome to the Sensu Documentation! This is a work in progress as we migrate from our previous project, [sensu/sensu-docs](https://github.com/sensu/sensu-docs-site).
 
-    - [Getting Started with Hugo](#getting-started-with-hugo)
+## Status
 
-    - [Deploying to Heroku](#deploying-to-heroku)
+- Pages from site are officially merged into the new docs site: https://github.com/sensu/sensu-docs-site, which is [tracked on these Milestones](https://github.com/sensu/sensu-docs-site/milestones)
+- The sensuapp.org/docs pages will remain separate until the Launch milestone is hit, tracked here: https://github.com/sensu/sensu-docs-site/milestones
 
-2. [Working With Hugo](#working-with-hugo)
+## Why Change?
 
-    - [Adding Content](#adding-content)
+This new site is a place for:
 
-    - [Overriding the Theme](#overriding-the-theme)
+- Versioned documentation that's easy to navigate
+- MUCH FASTER load times
+- Full-text search 😍
+- Easier contribution experience running Hugo locally
 
-    - [Understanding Front Matter](#understanding-front-matter)
+# Running Sensu Docs
 
-    - [Variables and Accessing Front Matter](#variables-and-accessing-front-matter)
-
-    - [Search](#search)
-
-## Project Setup
+Now that our project can be run locally with Hugo, there's some new steps to learn. Don't worry, we've documented how to do so:
 
 ### Getting Started with Hugo
```

Comment:
Can we please replace platform specific setup instructions with a link to https://gohugo.io/getting-started/installing or similar? I don't think we benefit from duplicating that sort of information here.

Reply:
I haven't dug into the nuances of the setup so I don't want to delete anything until I know we have it covered elsewhere. Issue #144 is intended to resolve this.